Add Arch Linux (Pacman) package support

ledbettj opened the pull request:
fpm already supports generating pacman packages, so just requires adding a task to build it.

```bash
$ export XZ_DEFAULTS="-T0"
$ gulp deploy:linux-x64:pacman
...
Created archive (/home/john/Projects/yakyak/dist/yakyak-1.5.5-linux-x64.tar.gz)
[16:45:34] Finished 'deploy:linux-x64:nodep' after 18 s
[16:45:34] Finished 'deploy:linux-x64' after 24 s
[16:45:34] Starting 'deploy:linux-x64:pacman:nodep'...
[16:45:35] Finished 'deploy:linux-x64:pacman:nodep' after 614 ms
[16:45:35] Finished 'deploy:linux-x64:pacman' after 24 s
Created pacman (yakyak-VERSION-linux-ARCH.tar.gz)
Done in 43.95s.

$ cd dist/ && sudo pacman -U yakyak-1.5.5-linux-x86_64.tar.gz 
loading packages...
warning: yakyak-1.5.5-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Packages (1) yakyak-1.5.5-1

Total Installed Size:  239.48 MiB
Net Upgrade Size:        0.00 MiB

:: Proceed with installation? [Y/n] 
```

averissimo commented:
Does a pacman have a specific extension?

I'm worried that when publishing to releases it might be a bit confusing.

I suggest adding "-pacman" before the extension. Does it seem ok?

See the `pacman` branch on yakyak where I've implemented this and added .travis support to build pacman automatically

ledbettj commented:
> Does a pacman have a specific extension?
> I'm worried that when publishing to releases it might be a bit confusing.

Can definitely see that.  Pacman does not have a specific extension, I think it in fact supports multiple compression formats and just expects the package to contain the appropriate files.

> 
> I suggest adding "-pacman" before the extension. Does it seem ok?
> 
> See the `pacman` branch on yakyak where I've implemented this and added .travis support to build pacman automatically

Yes, that looks good to me!
